test/lib/util/package_clause_generator_test.rb

```diff
@@ -113,102 +113,104 @@ def test_errata_ids
       goo_rule = FactoryBot.create(:katello_content_view_erratum_filter_rule,
                                     :filter => @filter, :errata_id => "Foo2")
 
-      expected_errata = [{"id" => {"$in" => ["Foo1", "Foo2"]}}]
+      expected_errata = [erratum_arel[:errata_id].in(["Foo1", "Foo2"])]
```

Comment:
I didn't really review the test code since the tests were failing, but you can also apply the comments from above here, but it's not as important to be readable (but still useful).